### DISCUSSION TITLE
Support for other exporters

### DISCUSSION THREAD

**Discussion by vasigorc** [ScalaConsultants/mesmer#472] [Q&A] [answered]
Hello 👋 

In your documentation you're giving an example of a [Prometheus set-up](https://mesmer.io/docs/getting-started). In our set-up we are using [OTLP (default)](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#otlp-exporter-span-metric-and-log-exporters). Is Prometheus a hard dependency?

Thank you!

**Comment by lgajowy** [comment, marked as answer]
Hi. 

No - you can still use otlp or any other opentelemetry exporter. We don't do anything prometheus specific. :) 

(The webpage is currently a wip so expect it to change)

**Comment by jczuchnowski** [comment]
Yes, sorry for the confusion @vasigorc . As @lgajowy said you can still use OTLP exporter. 
I will update the documentation to better reflect the different setup options.

**Comment by vasigorc** [comment]
Thank you Lukasz and Jakub!
